test/compilable/ddoc_markdown_code.d

```diff
@@ -0,0 +1,45 @@
+// PERMUTE_ARGS:
+// REQUIRED_ARGS: -D -Dd${RESULTS_DIR}/compilable -o- -preview=markdown
+// POST_SCRIPT: compilable/extra-files/ddocAny-postscript.sh
```

Comment:
Small nit: All tests should have a `TEST_OUTPUT` section as this helps to catch slight changes in behavior.

Reply:
I didn't know that—added one.